core/domain/exp_services_test.py

```diff
@@ -2856,3 +2856,23 @@ def test_draft_discarded(self):
         self.assertIsNone(exp_user_data.draft_change_list)
         self.assertIsNone(exp_user_data.draft_change_list_last_updated)
         self.assertIsNone(exp_user_data.draft_change_list_exp_version)
+
+
+class GetExplorationAndExplorationRightsTest(ExplorationServicesUnitTests):
+
+    def test_get_exploration_and_exploration_rights_object(self):
+        exploration_id = self.EXP_ID
+        self.save_new_valid_exploration(
+            exploration_id, self.owner_id, objective='The objective')
+
+        (exp, exp_rights) = (
+            exp_services.get_exploration_and_exploration_rights_by_id(
+                exploration_id))
+        self.assertEqual(exp.id, exploration_id)
+        self.assertEqual(exp_rights.id, exploration_id)
```

Comment:
Similarly, maybe we should test a few more fields of the exploration/exp_rights domain objects?

Reply:
same as above.